uktrade/data-hub-frontend
# feature/CLS2-460-add-adviser-to-task-form

## Description of change

- Add a new form field to the task form to allow assigning to other advisers

## Test instructions

Either add a new task, or edit an existing task. You will now see the form field to allow you to either assign the task to yourself or another adviser

## Screenshots

### Before

![image](https://github.com/uktrade/data-hub-frontend/assets/102232401/a17d68a1-2728-4e02-b0e6-032e1141b098)

### After

![image](https://github.com/uktrade/data-hub-frontend/assets/102232401/0e069198-3d7b-4f85-ba62-27867416a58e)

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
